### DISCUSSION TITLE
Does uAssets Annoyances block sign-up/log-in tooltips?

### DISCUSSION THREAD

**Discussion by wavygecko** [uBlockOrigin/uAssets#14746] [Q&A] [answered]
Is blocking tooltip popups asking users to sign up or log in something that uAssets Annoyances covers? e.g., https://github.com/easylist/easylist/issues/13231

**Comment by MasterKia** [comment]
Related: https://github.com/uBlockOrigin/uAssets/issues/12604#issuecomment-1088304068

**Reply by krystian3w** [reply]
#14727

**Comment by uBlock-user** [comment, marked as answer]
Yes, but on case-by-case basis.

**Comment by krystian3w** [comment]
https://github.com/uBlockOrigin/uAssets/issues/5911
